scipy/stats/tests/test_continuous_basic.py

```diff
@@ -662,3 +664,40 @@ def test_methods_with_lists(method, distname, args):
                             [f(*v) for v in zip(x, *shape2, loc, scale)],
                             rtol=1e-15, atol=1e-15)
 
+
+@pytest.mark.parametrize('distname, args', distcont)
```

Comment:
So only continuous distributions are tested? Since there are similar code paths in `rv_discrete`, these should be tested, too.

Reply:
it seems that `check_pickling` [is also used for testing rv_discrete](https://github.com/scipy/scipy/blob/master/scipy/stats/tests/test_discrete_basic.py#L57).